test/counter.spec.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { shallow, mount } from 'enzyme';
+import { Counter } from '../src/counter'
+
+test('renders without crashing', () => {
+  mount(<Counter />);
+})
+
+test('renders a button component', () => {
+  const counter = shallow(<Counter />);
```

Comment:
This line is repeated in each of the three tests. You can move it into a `beforeEach` block